test/Microsoft.AspNet.Routing.Tests/BuilderExtensionsTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Builder;
+using Moq;
+using Xunit;
+
+namespace Microsoft.AspNet.Routing.Tests
```

Comment:
`Microsoft.AspNet.Builder` to match `BuilderExtensions` (plus: almost never `...Test` or `...Tests`)